frontend/common/kubeflow-common-lib/projects/kubeflow/src/lib/form/name-namespace-inputs/name-input/name-input.component.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
 
 import { NameInputComponent } from './name-input.component';
-import { FormControl } from '@angular/forms';
+import { UntypedFormControl } from '@angular/forms';
```

Comment:
did the typed form produce issues? otherwise we can stick to typed forms https://angular.io/guide/typed-forms#overview-of-typed-forms

Reply:
This was done automatically when upgrading to angular 14 https://angular.io/guide/typed-forms#automated-untyped-forms-migration. But i agree that we should stick to typed forms to be more robust. I can create an issue to do the transition to typed forms